bin/apply.bash

```diff
@@ -47,6 +47,17 @@ apps_apply() {
   log_info "---"
   log_info "Start Apps ${action} on ${2/_/ }"
 
+  validate_version "${1}"
+  if get_migration_status "${1}" &>/dev/null; then
+    log_fatal "Migration ongoing, try again when it has completed or 'ck8s upgrade unlock'"
+  fi
```

Comment:
TODO move into function and call before `apps_apply()`

Reply:
Turns out to need a larger refactor to allow than what seems in scope for this PR, changing to using a variable instead to make tests faster